static/app/gettingStartedDocs/javascript/javascript.tsx

```diff
@@ -46,7 +46,7 @@ ${getFeedbackConfigOptions(params.feedbackOptions)}}),`
   }${
     params.isReplaySelected
       ? `
-        Sentry.replayIntegration(${getReplayConfigOptions(params.replayOptions)}),`
+        Sentry.replayIntegration(${getReplayConfigOptions(params.replayOptions ?? {block: true, mask: true})}),`
```

Comment:
I would rather pass it in `https://github.com/getsentry/sentry/blob/master/static/app/components/onboarding/gettingStartedDoc/onboardingLayout.tsx`, then it is done at a central point for all platforms during creation.

Reply:
good catch @ArthurKnaus! fixed